src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = 40;
+
+  if (days >= 7) {
+    totalAmount = totalAmount * days - 50;
+  } else if (days >= 3) {
+    totalAmount = totalAmount * days - 20;
+  } else {
+    totalAmount = totalAmount * days;
+  }
```

Comment:
what is `50` and `20`? add some meaningful constants
```suggestion
  if (days >= 7) {
    return totalAmount * days - 50;
  } 
  
  if (days >= 3) {
    return totalAmount * days - 20;
  }
  
  return totalAmount * days;
```